Detections/Syslog/ssh_potentialBruteForce.yaml

```
@@ -1,23 +1,30 @@
 ﻿id: e1ce0eab-10d1-4aae-863f-9a383345ba88
 name: ssh Potential Brute Force
-description: an IP address that had 15 failed attempts to sign in via ssh in a 4 hour block during a 24 hour time period
+description: |
+  'Identifies an IP address that had 15 failed attempts to sign in via ssh in a 4 hour block during a 24 hour time period.'
 severity: Medium
 requiredDataConnectors:
   - connectorId: Syslog
     dataTypes:
       - Syslog
-queryFrequency: 24h with 4h bins
-queryPeriod: 24h
+queryFrequency: 1d
+queryPeriod: 1d
 triggerOperator: gt
 triggerThreshold: 0
 tactics:
-  - InitialAccess
+  - CredentialAccess
+relevantTechniques:
+  - T1110
 query: |
 
+  let timeframe = 1d;
+  let threshold = 15;
   Syslog
+  | where TimeGenerated >= ago(timeframe)
   | where SyslogMessage contains "Failed password for invalid user"
   | where SyslogMessage contains "ssh2" 
   | parse kind=relaxed SyslogMessage with * "invalid user" user " from " ip " port" port " ssh2"
   | project user, ip, port, SyslogMessage, EventTime
   | summarize count() by ip, bin(EventTime, 4h)
```

maybe include the user info? 
```suggestion
  | summarize count(), makeset(user) by ip, bin(EventTime, 4h)
```